HelloWorld.java

```diff
@@ -1,5 +1,6 @@
 public class HelloWorld {
     public static void main(String[] args) {
+        // ddd
         System.out.println("Hello, World!");
```

Comment:
Here is the explanation
and here is the suggestion

```suggestion
        System.out.println("Hello, World!");
```